src/backend/tools/google_drive/auth.py

```diff
@@ -118,13 +78,15 @@ def try_refresh_token(
             return False
         tool_auth_crud.update_tool_auth(
             session,
-            ToolAuth(
+            tool_auth,
+            UpdateToolAuth(
```

Comment:
@BeatrixCohere follow up from your Google Drive PR, this addition is important for the `try_refresh_token` method. `update_tool_auth` expects a `BaseModel` class that inherits the `model_dump()` method https://github.com/cohere-ai/cohere-toolkit/blob/main/src/backend/crud/tool_auth.py#L58